src/libostree/ostree-sysroot-deploy.c

```diff
@@ -2135,8 +2131,7 @@ ostree_sysroot_deploy_tree (OstreeSysroot     *self,
         goto out;
     }
 
-  { ostree_cleanup_sepolicy_fscreatecon gpointer dummy = NULL;
-
+  {
```

Comment:
This *is* used to unset the context state.   We likely need to annotate `G_GNUC_USED` or so.